docs/Getting Started/Recurring Tasks.md

```diff
@@ -62,11 +62,11 @@ Use this setting to control where the recurring task is inserted. The default is
 When you create a recurring task, you can decide whether the next occurrence should be based on the original dates or the date when you completed the task.
 The default behavior results in newly created tasks having dates relative to the original task rather than "today".
 
-For example, given that today is the 13. February 2022 and you just completed the lower task:
+For example, given that today is the 13 February 2022 and you just completed the lower task:
 
 ```markdown
-- [ ] sweep the floors 🔁 every week ⏳ 2021-02-13
-- [x] sweep the floors 🔁 every week ⏳ 2021-02-06 ✅ 2022-02-13
+- [ ] sweep the floors 🔁 every week ⏳ 2022-02-13
+- [x] sweep the floors 🔁 every week ⏳ 2022-02-06 ✅ 2022-02-13
```

Comment:
As noted by the surrounding text, the completion should be one week after the originally scheduled date (not one week and a year), and the scheduled date of the new task should be the same as the completed date of the previous occurrence